pims/matplotlib.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import (absolute_import, division, print_function,
```

Comment:
shadowing `matplotlib` is probably technically ok, but I think is bad style.

I would also import mpl at the top of this file and not import these as part of the api (or put this import in a try-except)